upgrading.adoc

```diff
@@ -287,19 +288,19 @@ as Bitcoin continues to grow.
 ____
 
 [[taproot-deployment]]
-==== Taproot upgrade - Speedy trial
+==== Taproot upgrade - Speedy Trial
 
-When Taproot was ready for deployment, meaning all technical details
-around its consensus rules were implemented and had reached broad
-approval from the community, discussions on how to actually deploy it
+When Taproot was ready for deployment [comm: add time coordinates], meaning all the technical details
+around its consensus rules had been implemented and had reached broad
+approval within the community, discussions on how to actually deploy it
 started to heat up. These discussions had been pretty low key up until
-this point.
+that point.
 
-Lot's of activation mechanism proposals started floating around and
+Lots of proposals for activation mechanisms started floating around, and
 David Harding
 https://en.bitcoin.it/wiki/Taproot_activation_proposals[summarized
-them on the Bitcoin Wiki]. In that article he explains some properties
-of BIP8 which at that time had some recent changes made to make it
+them on the Bitcoin Wiki]. In his article he explained some properties
+of BIP8, which at that time had some recent changes made in order to make it
 more flexible.
 
 ____
```

Comment:
Missing quote info: 

[quote, David Harding, Taproot Activation Proposals on the Bitcoin Wiki (2020)]